android/src/main/java/com/rudderstack/android/utilities/AppVersion.kt

```diff
@@ -0,0 +1,22 @@
+package com.rudderstack.android.utilities
+
+data class AppVersion(
+    val previousBuild: Int,
+    val previousVersionName: String,
+    val currentBuild: Int,
+    val currentVersionName: String,
+) {
+
+    companion object {
+        const val DEFAULT_BUILD = -1
+        const val DEFAULT_VERSION_NAME = ""
+    }
+
+    fun isApplicationInstalled(): Boolean {
```

Comment:
data classes should not contain any business logic. 
Move the AppVersion data class to model module.